src/sentry/backup/imports.py

```diff
@@ -135,6 +135,34 @@ def _import(
         if decryptor is not None
         else src.read().decode("utf-8")
     )
+
+    # We have to be careful when removing fields from our model schemas, since exports created using
+    # the old-but-still-in-the-support-window versions could have those fields set in the data they
+    # provide. This dict serves as a map of all fields that have been deleted on HEAD but are still
+    # valid in at least one of the versions we support. For example, since our current version
+    # support window is two minor versions back, if we delete a field at version 24.5.N, we must
+    # include an entry in this map for that field until that version is out of the support window
+    # (in this case, we can remove shim once version 24.7.0 is released).
+    #
+    # NOTE TO FUTURE EDITORS: please keep the `deleted_fields` dict, and the subsequent `if` clause,
+    # around even if the dict is empty, to ensure that there is a ready place to pop shims into. For
+    # each entry in this dict, please leave a TODO comment pointed to a github issue for removing
+    # the shim, noting in the comment which self-hosted release will trigger the removal.
+    deleted_fields: dict[str, set[str]] = {}
```

Comment:
Is `deleted_fields` being updated over time to store all the deleted fields? If so, would it be better to place this as a global variable at the top of the file or somewhere more clear?